airflow-core/src/airflow/ui/src/pages/DagsList/AssetSchedule.tsx

```diff
@@ -63,7 +63,16 @@ export const AssetSchedule = ({ assetExpression, dagId, latestRunAfter, timetabl
     return (
       <HStack>
         <FiDatabase style={{ display: "inline" }} />
-        <Link asChild color="fg.info" display="block" fontSize="sm">
+        <Link
+          asChild
+          color="fg.info"
+          display="block"
+          fontSize="sm"
+          maxWidth="200px"
+          overflow="hidden"
+          textOverflow="ellipsis"
+          whiteSpace="nowrap"
```

Comment:
```suggestion
          truncate
```
It could be clearer in this way
ref: https://tailwindcss.com/docs/text-overflow